### PR TITLE
[DOCS] Adds security privilege info to inference bucket aggregation

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds information about the required security privileges to the inference bucket aggregation documentation.

### Preview

[Inference bucket aggregation](https://elasticsearch_59604.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/search-aggregations-pipeline-inference-bucket-aggregation.html)

### CODE REVIEW

File: docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc [hunk 1]
```diff
@@ -1,10 +1,13 @@
 [role="xpack"]
 [testenv="basic"]
 [[search-aggregations-pipeline-inference-bucket-aggregation]]
-=== Inference Bucket Aggregation
+=== {infer-cap} Bucket Aggregation
 
-A parent pipeline aggregation which loads a pre-trained model and performs inference on the
-collated result field from the parent bucket aggregation.
+A parent pipeline aggregation which loads a pre-trained model and performs 
+{infer} on the collated result field from the parent bucket aggregation.
```
Comment (davidkyle) [inline docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc:7]: Should field be plural? Reading this back I'm not sure

on the collated result field**s**

**Review by davidkyle — APPROVED** [review]
LGTM

File: docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc (hunk 1 above)
Comment (szabosteve) [inline docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc:7]:
```suggestion
{infer} on the collated result fields from the parent bucket aggregation.
```

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: 35512a9
7.9: 206b2a8

**Pull request merged**